NLog.Web.AspNetCore/LayoutRenderers/AspNetRequestFormLayoutRenderer.cs

```diff
@@ -0,0 +1,88 @@
+﻿#if ASP_NET_CORE
+using Microsoft.Extensions.Primitives;
+#endif
+using NLog.LayoutRenderers;
+using NLog.Web.Internal;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NLog.Web.LayoutRenderers
+{
+    /// <summary>
+    /// ASP.NET Request Form Data
+    /// </summary>
+    /// <example>
+    /// <para>Example usage of ${aspnet-request-form}:</para>
+    /// <code lang="NLog Layout Renderer">
+    /// ${aspnet-request-form} - Produces - All Form Data from the Request with each key/value pair separated by a new line.
+    /// ${aspnet-request-form:Include=id,name} - Produces - Only Form Data from the Request with keys "id" and "name".
+    /// ${aspnet-request-form:Exclude=id,name} - Produces - All Form Data from the Request except the keys "id" and "name".
+    /// ${aspnet-request-form:Separator= / } - Produces - All Form Data from the Request with each key/value pair separated by " / ".
+    /// </code>
+    /// </example>
+    [LayoutRenderer("aspnet-request-form")]
+    public class AspNetRequestFormLayoutRenderer : AspNetLayoutRendererBase
+    {
+        /// <summary>
+        /// Gets or sets the form keys to include in the output.  If omitted, all are included.
+        /// </summary>
+        /// <docgen category='Rendering Options' order='10' />
+        public string Include { get; set; }
```

Comment:
You could use list/set here :)

(Since NLog 4.4 - https://nlog-project.org/2016/12/14/nlog-4-4-is-live.html)

I prefer a `ISet<String>` (for .NET 3.5 is should be `HashSet<String>`)

You could make it default case insensitive.

Reply:
Something like this?
```C#
#if ASP_NET_CORE
        public ISet<string> Include { get; set; }
#else
        public HashSet<string> Include { get; set; }
#endif
```

Reply:
yes indeed 👍

Reply:
Changed Include and Exclude to case insensitive HashSets.
https://github.com/DrewBrasher/NLog.Web/commit/8eb037049c212c70d2b85668883788d0565d5bb4